.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.19.x, 1.20.x]
+        go-version: [1.20.x, 1.21.x]
```

Comment:
Isn't it a tad weird that we support the last 3 stable Go versions, but only test 2?

Reply:
Agreed - just expanded it to the three versions we support. PTAL